app/src/redux/config/schema-types.ts

```diff
@@ -18,6 +18,7 @@ export type DevInternalFlag =
   | 'reactScan'
   | 'quickTransferExportPython'
   | 'camera'
+  | 'protocolContentsLog'
```

Comment:
nit: same here, can you mention that it is for quick transfer? like:
```suggestion
  | 'protocolContentsLogQT'
```
or 
```suggestion
  | 'quickTransferProtocolContentsLog'
```